### PR TITLE
[IMP] mail: turn 'MessageView/authorAvatarTitleText' into field

### PR DESCRIPTION
**Pull request by alexkuhn**
Task-2793280

### CODE REVIEW

File: addons/mail/static/src/models/message_view/message_view.js [hunk 1]
```diff
@@ -148,6 +155,9 @@ registerModel({
             isCausal: true,
             readonly: true,
         }),
+        authorOpenChatText: attr({
```
Comment (oomsveta) [inline addons/mail/static/src/models/message_view/message_view.js:158]: I don't think having 'author' in the name is relevant here. It confused me at least.

Comment (alexkuhn) [reply addons/mail/static/src/models/message_view/message_view.js:158]: It's important to know it's in the "author" part of the message view that the text is shown.
I'll just rename the field to `authorTitleText`.

Comment (oomsveta) [reply addons/mail/static/src/models/message_view/message_view.js]: How about `authorAvatarTitleText` then? What confuses me about `author` is that you can't guess exactly what it is in the interface without reading the code.

Apart from that, `TitleText` feels much more meaningful to me, good improvement 👍️

**Review by oomsveta — APPROVED** [review]

**Comment by alexkuhn** [comment]
@robodoo r+

**Pull request closed without merging**